### ISSUE TITLE
Can it be run on Windows?

### ISSUE THREAD

**Issue by ZeusFunk** [e4s2022/e4s#15]
I hope to be able to run it on Windows. I haven't tried it yet and I'm not sure if it can be successful.

**Comment by Inferencer** [comment]
> I hope to be able to run it on Windows. I haven't tried it yet and I'm not sure if it can be successful.

yes, use this fork https://github.com/justinjohn0306/e4s-windows

**Comment by aprillysea** [comment]
can you run it？
